gapis/trace/android/trace.go

```diff
@@ -283,6 +283,10 @@ func (t *androidTracer) TraceConfiguration(ctx context.Context) (*service.Device
 	*/
 	if len(t.b.Instance().GetConfiguration().GetDrivers().GetVulkan().GetPhysicalDevices()) > 0 && *flags.EnableVulkanTracing {
 		apis = append(apis, tracer.VulkanTraceOptions())
+		// If ANGLE is enabled and available, need to also append ANGLE trace mode
```

Comment:
Do we want to put ANGLE flag behind vulkan tracing flag?

Reply:
ANGLE can currently only be used for Graphics tracing mode so this makes sense to me. If there comes a point when we want to enable ANGLE or Vulkan tracing by default without the other, we would have to refactor this.